migration/20181108-remove-bad-content-cafe-covers.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+#
+# Stop works from considering covers from Content Cafe which turned out to be
+# placeholders.
+
+import os
+import sys
+bin_dir = os.path.split(__file__)[0]
+package_dir = os.path.join(bin_dir, "..")
+sys.path.append(os.path.abspath(package_dir))
+
+from core.s3 import *
+from core.config import Configuration
+from core.model import (
+    production_session,
+    Representation,
+    get_one,
+    PresentationCalculationPolicy,
+)
+
+_db = production_session()
+qu = _db.query(Representation).filter(
+    Representation.image_height==120).filter(
+        Representation.image_width==80).filter(
+            Representation.url.like("http://contentcafe2.btol.com/%")
+        )
+policy = PresentationCalculationPolicy.recalculate_everything()
+for rep in qu:
+    identifiers = [h.identifier for h in rep.resource.links]
+    fix_editions = []
+    for identifier in identifiers:
+        for edition in identifier.primarily_identifies:
+            if 'Content' in edition.cover_thumbnail_url or 'Content' in edition.cover_full_url:
```

Comment:
This check is necessary because sometimes the work ends up with a cover image from a more reliable source like Overdrive.